lib/templates/stalePullRequest.hbs

```diff
@@ -0,0 +1,7 @@
+Bump! 
```

Comment:
I think `Bump!` may not be the _most_ polite thing here. :smirk: 

May leave the `Thanks!`?

Also just a thought- Do we also want to make the username of the user who opened the PR available to the PR template? This way we can @ mentioned them specifically and ask them to take action (since ultimately it's the responsibility of the user who opened the PR to get it merged).

Reply:
I just feel like the concierge constantly starting every comment with "thanks" makes it start to lose its meaning. 

I changed it though, and added a `userName` parameter!